app/assets/stylesheets/dm/components/_links.scss

```diff
@@ -29,4 +32,45 @@
       text-decoration: underline;
     }
   }
+}
+
+.dm-primary-link {
+  color: color($theme-link-color) !important;
```

Comment:
we can keep this but just fyi you can also do `@include u-text($theme-link-color, !important);` if it is not a custom color and we don't need the `important!` we can do `  @include u-text('primary-vivid');` :D

Reply:
I added the mixin here 😃

Reply:
I'm not seeing that change haha but this is all good regardless 😅 Was trying to share another way to change the color of the text using uswds mixins :)